lib/algolia/config/recommendation_config.rb

```diff
@@ -1,5 +1,6 @@
 module Algolia
   module Recommendation
+    # <b>DEPRECATED:</b> Please use <tt>Algolia::Personalization::Config</tt> instead.
     class Config < BaseConfig
```

Comment:
Is there a reason you didn't go for the same solution as the client? extend the Personalization version and remove all content in this class?
```suggestion
    class Config < Algolia::Personalization::Config
```